### ISSUE TITLE
Cannot create a quickstart while another quickstart is being created

### ISSUE THREAD

**Issue by mauromorales** [jenkins-x/jx#6270]
### Summary 

When trying to create a quickstart while another quickstart is being created (pipelines haven't finished), the command errors in an unfinished state.

### Steps to reproduce the behavior

1. Run `jx quickstart` and follow the quickstart setup
2. Before the execution of the pipelines for the previous command finish running, run `jx quickstart` again with a new app name and the same user and owner than for previous quickstart

### Expected behavior

Create a second quickstart

### Actual behavior

The second quickstart returns the following error:

```
error: failed to create Pull Request on the development environment git repository https://github.com/mauromorales/environment-mm-dev-dev.git: unable to create pull request: failed to create PR for base master and head branch bump-resource-version-34453d54-14fe-11ea-a4e6-acde48001122 from temp dir /var/folders/46/3srjnjn16mq8ldbk4q1qwwq40000gn/T/create-pr951443946: fetching pull/1/head:df43fc3a-76e9-4bde-aed4-0d3be37834b7 for merge: git output: remote: Repository not found.
fatal: repository 'https://github.com/mauromorales/environment-mm-dev-dev.git/' not found: failed to run 'git fetch origin pull/1/head:df43fc3a-76e9-4bde-aed4-0d3be37834b7' command in directory '/var/folders/46/3srjnjn16mq8ldbk4q1qwwq40000gn/T/create-pr951443946', output: 'remote: Repository not found.
fatal: repository 'https://github.com/mauromorales/environment-mm-dev-dev.git/' not found'
```

### Jx version

The output of `jx version` is:

```
mrls@S2N jx-qs-test-1 % jx version
NAME               VERSION
jx                 2.0.441-dev+225639507
Kubernetes cluster v1.13.11-gke.14
kubectl            v1.16.2
helm client        Client: v2.14.3+g0e7f3b6
git                2.21.0 (Apple Git-122.2)
Operating System   Mac OS X 10.15.1 build 19B88
```

### Kubernetes cluster

GKE following the install & separated boot setup

### Operating system / Environment

macOS

**Comment by abayer** [comment]
Huh. I’m betting this is related to changing the creds used for creating quick starts in #6199. Not sure off the top of my head why that would break the dev env PR for a second quick start but not the first, though.

**Comment by mauromorales** [comment]
In the same environment i was working on I decided to make the dev env private again and test it out, so far I've noticed that again the first run works. Immediately after, while pipelines are still running I decided to try it again and it broke. I decided to wait until the pipelines where over and test again this time it worked fine. Tried this a copule of times and my conclusion so far is that while the first quickstart is being created, no additional quickstarts can be created (changing title).

My assumption right now is that the dev environment resource (or something related to it) is in a tainted/running state which doesn't allow a new change to happen in the dev environment until it is back to a clean/done state. I don't know if this is per user or for the entire environment, in case of the latter then IMO this is pretty bad because the number of issues can increase depending on the number of users in the system.

I haven't investigated enough to figure out what it takes to solve the issue but If we cannot get away from it in the short term, we should at least clean up the quickstart repositories that were created after a failure to incorporate them to the dev environment, otherwise it's a bit annoying to have to go and clean that up manually in order to reuse the repository name.
